game_event/urls.py

```diff
@@ -5,4 +5,8 @@
 urlpatterns = [
     path('create/', views.display_game_event_form, name='display game event form'),
     path('create/process', views.process_game_event_form, name='process game event form'),
-    path('', views.game_events, name='game-events')]
+    path('', views.game_events, name='game-events'),
+    path('<id>/', views.game_event, name='game_event'),
+    path('join-event/<id>/', views.join_event, name='join_event'),
+    path('process-answer/<id>/', views.process_answer, name='process_answer'),
```

Comment:
I think we need a more informative name than process_answer. It's not self explanatory enough (which answer?). Also, suppose tomorrow there will be yet another question, we'd need to somehow differentiate those (`process_answer_2` will it be? :) )